File: review.py
# Variables
name = "Dhari"
age = 24
height = 174.5
likesPizza = True

print(name)

name = "Alenezi"

print(name)


# Conditionals
grade = 80

if grade >= 90:
    print('A')
elif grade >= 80:
    print('B')
elif grade >= 70:
    print('C')
elif grade >= 60:
    print('D')
else:
    print('F')

# Functions
def printBio(name, age):
    print(f"Name: {name} , Age: ${age}")

def sum(num1, num2):
    return num1 + num2

# Lists
studentList = ["Dhari", "Ahmed", "Mohammed"]
print(studentList[0])
print(len(studentList))


# Loops
for student in studentList:
    print(student)


# Dictionaries
person = { 
   "name": "Dhari",
   "age": 24,
   "height": 174,
}

# Keys not index
print(person['name'])
print(person['age'])

File: review.dart
void main() {
  String name = 'Dhari';
  int age = 24;
  double height = 174.5;
  bool likesPizza = true;

  print(name);

  name = "Alenezi";

  print(name);

  double grade = 80;
  if (grade >= 90) {
    print('A');
  } else if (grade >= 80) {
    print('B');
  } else if (grade >= 70) {
    print('D');
  } else {
    print('f');
  }

  // Function
  void printBio(String name, int age) {
    print('Name: ${name} , Age: ${age}');
  }

  double sum(double num1, double num2) {
    return num1 + num2;
  }

  print(sum(10, 20));

  List<String> studentList = ["Dhari", "Ahmed", "Mohammed"];

  for (String student in studentList) {
    print(student);
  }

  studentList.forEach((student) {
    print(student);
  });

  Map<String, dynamic> person = {
    'name': 'Dhari',
    'age': 24,
    'height': 174,
  };

  print(person['name']);
}
